src/main/kotlin/dev/anvith/alphabet/GenerateIntentionColor.kt

```diff
@@ -22,29 +26,30 @@ class GenerateIntentionColor : PsiElementBaseIntentionAction(), IntentionAction
         return NAME
     }
 
-
     override fun isAvailable(project: Project, editor: Editor?, element: PsiElement): Boolean {
         return element is XmlToken && element.parents.any { it is XmlTag && it.name == COLOR_TAG && it.parentTag?.name == RES_TAG }
     }
 
     override fun invoke(project: Project, editor: Editor?, element: PsiElement) {
         val colorTag = element.parents.first { it is XmlTag && it.name == COLOR_TAG }
         val percentageString = Messages.showInputDialog(project, PERCENTAGE, NAME, null)
-
+        val percentages = percentageString?.trim()?.split(",")
         val modifiedColorTag = colorTag.copy() as XmlTag
         val attribute = modifiedColorTag.children.first { it is XmlAttribute && it.name == NAME_ATTR } as XmlAttribute
         val colorValue = modifiedColorTag.children.first { it is XmlText && it.value.startsWith("#") } as XmlText
 
-        addAlphaVariant(project, modifiedColorTag, attribute, colorTag, colorValue, percentageString)
+        percentages?.forEach {
```

Comment:
consider reversing the strings before adding to maintain expected order by user
-> `percentages.reversed().forEach {`